src/applications/static-pages/social-share-links.js

```diff
@@ -27,10 +28,12 @@ export function openShareLink() {
             text: metaDescription,
             url: metaUrl,
           })
-          .then(() => event.preventDefault())
-          .catch(() => document.location.assign(link.href));
+          .catch(err => {
+            // eslint-disable-next-line no-console
+            console.error('Share failed', err);
```

Comment:
Are you adding this for local debugging? We have a way of logging errors if you think this is something that we can take action to fix when it happens.